lfsccm/lfsccm/main.py

```diff
@@ -219,12 +222,13 @@ def _expand_files(self, files):
             is_recursive = False
             if recursive == 'r':
                 is_recursive = True
-                filepath = re.sub('\*+', '**', filepath)
+                filepath = re.sub('\\*+', '**', filepath)
```

Comment:
This might be invalid fix so if @kaneko-tx or @rkmrHonjo knows the correct thing. Please let me know to fix it.

Reply:
I got the same result both before and after the fix.
```
>>> re.sub('\*+', '**', '/path/to/dir/*')
'/path/to/dir/**'

>>> re.sub('\\*+', '**', '/path/to/dir/*') 
'/path/to/dir/**'
```
However, looking at docs[^1], it seems that two backslashes are better when raw string notation is not used.
So I think your fix is good.

[^1]: https://docs.python.org/3/library/re.html#raw-string-notation